### PR TITLE
Remove version number from system test prereq jar references in getDependency script

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
- Removes version number from all prereq jar references in getDependency script 
- Related to : https://github.com/AdoptOpenJDK/stf/issues/98
 
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Test: Internal Grinder/13187

File: buildenv/jenkins/getDependency [hunk 1]
```diff
@@ -22,17 +22,17 @@ def testBuild() {
 				sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
 				sh '${WORKSPACE}/j2sdk-image/jre/bin/java -version'
 				sh 'git clone https://github.com/AdoptOpenJDK/openjdk-systemtest'
-				sh 'git clone https://github.com/AdoptOpenJDK/stf'
+				sh 'git clone https://github.com/Mesbah-Alam/stf -b consolidate-prereq-version-numbers'
```
Comment (llxia) [inline buildenv/jenkins/getDependency:25]: Please remove the personal repo and branch.

**Comment by llxia** [comment]
This PR is related to 3 other PRs. They should be merged together.
https://github.com/AdoptOpenJDK/stf/pull/99
https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/392
https://github.com/eclipse/openj9-systemtest/pull/123

File: buildenv/jenkins/getDependency (hunk 1 above)
Comment (Mesbah-Alam) [reply buildenv/jenkins/getDependency]: Ah. That one was missed! Thanks for the catch! Updated.

**Comment by smlambert** [comment]
Since these changes are an enhancement (and not needed to fix a defect), we will defer merging this until after the release that is happening this week.  Thanks for this work @Mesbah-Alam !

**Comment by Mesbah-Alam** [comment]
Yes, that makes sense, Shelley. Thanks! @ShelleyLambert

**Review by smlambert — APPROVED** [review]

**Comment by smlambert** [comment]
@llxia - if you approve of these changes, I will merge all 4 related PRs together to usher this nice enhancement from @Mesbah-Alam in.

**Review by llxia — APPROVED** [review]
LGTM

**Comment by Mesbah-Alam** [comment]
Thanks @llxia and @ShelleyLambert for reviewing and merging this. 

Just a heads up, there is one more set of review requests, consisting of 4 PRs, heading your way, soon - which has to do with switching iteration-based load tests to time-based. I am currently running some testing on the changes.

**Comment by smlambert** [comment]
Wonderful, thanks @Mesbah-Alam for all of this work on improvement and make them easier to maintain!

**Comment by llxia** [comment]
https://ci.eclipse.org/openj9/job/systemtest.getDependency/195/

**Comment by Mesbah-Alam** [comment]
Thanks @llxia for running the systemtest.getDependency build at ci.eclipse. I don't have permission to manually run the job there. I had run it at ci.Adopt, and the two internal Jenkins instances last night.

**Pull request merged**